app.js

```diff
@@ -0,0 +1,206 @@
+    description:
+      'The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.',
+    skills: ['ruby', 'rails', 'bootstrap'],
+    image: 'images/wishyacht.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Companies 3D',
+    info: ['frontend', 'Api', '2022'],
+    description:
+      "this is an informative/numeric web application that shows information about the canadian technical companies and their statistcs like the market cap and the trading status it also has search feature and for each company page it's unique root and responsive website",
+    skills: ['react', 'redux', 'bootstrap'],
+    image: 'images/company.png',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Space hub',
+    info: ['pair', 'APIs', '2022'],
+    description:
+      'A web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions',
+    skills: ['react', 'redux', 'bootstrap'],
+    image: 'images/space.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+];
+
+const btns = document.querySelectorAll('.work button');
+const bgModal = document.querySelector('.bg-modal');
+
+function popUp(index) {
+  const {
+    title, info, description, skills, image,
+  } = popupCards[index];
+
+  const infoHtml = info
+    .map(
+      (i) => `
+      <li class="canopy">${i}</li>
+      ${
+  i === info[info.length - 1]
+    ? ''
+    : '<li><img src="images/Counter.png" alt="counter" /></li>'
+}
+    `,
+    )
+    .join('');
+
+  const skillsHtml = skills
+    .map((s) => `<li class="highlight font-1">${s}</li>`)
+    .join('');
+
+  bgModal.innerHTML = `
+  <div class="modal-content">
+        <div class="modal-header">
+          <h3 class="tonic">${title}</h3>
+          <span class="close-button">+</span>
+        </div>
+        <ul class="ul-box1 flex0">
+          ${infoHtml}
+        </ul>
+        <a href="#"
+          ><img src="${image}" alt="Snapshoot-Portfolio" class="snapshot"
+        /></a>
+        <div class="flex-99">
+          <div class="flex-98">
+            <p class="description-2 flex0">
+            ${description}
+            </p>
+          </div>
+          <div>
+            <ul class="ul-box2 flex0">
+             ${skillsHtml}
+            </ul>
+            <hr class="solid-1">
+            <div class="buttons">
+              <button type="button" class="live-btn">
+                see live
+                <img src="images/Icon.png" alt="live"
+              </button>
+              <button type="button" class="live-btn">
+                see source
+                <img src="images/github.png" alt="live"
+              </button>
+            </div>
```

Comment:
As I have realized that some of the changes you introduced may have led to a distortion of the design, I have to mention them now.
- [ ] On mobile view, the icons distort and do not show correctly and this is affecting the look of the design.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/197112705-7b7ae37c-72c8-4755-9253-9898ddb6713a.png)

- [ ] Also,as requested by the [previous reviewer](https://github.com/EngruugbeJimmy/Portfolio-mobile-view/pull/5#discussion_r1000841826), kindly ensure that the button states i.e on hover, clicked active etc are implemented. If this might have been a challenge to implement, have a look at [this article](https://zellwk.com/blog/style-hover-focus-active-states/) to learn how to implement it.